home/tests/test_models.py

```diff
@@ -760,7 +760,6 @@ def test_template_create_with_buttons(self):
         wat.quick_replies.add(created_qr)
         wat.save()
         wat.save_revision()
-        wat.submit_whatsapp_template(None)
 
         request = responses.calls[0].request
         assert json.loads(request.body) == {
```

Comment:
Maybe we can also assert on that the correct status fields have been updates on the model?